src/components/dashboard/FeedItems/ImageUtils.web.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import SVG from 'react-inlinesvg'
+
+export const SvgImage = ({ src, width, height }) => <SVG src={src} width={width} height={height} />
+
+export const resolveAssetSource = base64 => {
```

Comment:
1. we **already** have image utils in `lib/...`. move it there (except SvgImage)
2. use some existing library for web
3. why this is needed ? if for the aspect ratio - we dont need it (pls read my comment below), revert